src/application/wizard/resolvers.ts

```diff
@@ -37,6 +39,12 @@ const resolvers = (wizard: WizardService, userService: UserService): IResolvers
             variables: { fileId: FileId; submissionId: SubmissionId },
             context,
         ): Promise<boolean> {
+            await pubsub.publish('UPLOAD_STATUS', {
+                userId: 'c0e64a86-2feb-435d-a40f-01f920334bc4',
```

Comment:
Hardcoded user id?

Reply:
Haha, good catch! Not sure that slipped through!!!!

Reply:
This is testing code. It shouldn't be there...

Reply:
Removed and pushed.